.github/workflows/prerelease.yml

```diff
@@ -74,7 +71,7 @@ jobs:
       UPLOAD_URL: ${{ steps.create_release.outputs.upload_url }}
       INTEGRATION_NAME: ${{ steps.check.outputs.NAME }}
       INTEGRATION_VERSION: ${{ steps.check.outputs.VERSION }}
-      RELEASE_TAG: ${{ steps.check.outputs.PACKAGE_NAME }}-${{ steps.check.outputs.VERSION }}
+      RELEASE_TAG: ${{ steps.check.outputs.PACKAGE_NAME }}-${{ steps.check.outputs.VERSION }}q
```

Comment:
Is this trailing `q` expected? 🤔

Reply:
Oops. I think it was a vim `:q` typo. Fixing it.

Reply:
Fixed in [#3d140d3](https://github.com/newrelic/newrelic-prometheus-exporters-packages/pull/133/commits/3d140d3fb93adf5e75ba32a64088c1b656641490)